doc/manuals/user/walkthrough_apiv1.md

```diff
@@ -1020,6 +1020,8 @@ Let's examine in detail the different elements included in the payload:
     notified of the value of *temperature* each time the value of
     *pressure* changes may not be too useful. The example is chosen this
     way only to show the enormous flexibility of subscriptions.
+-   You can leave empty the condValue list (or even omit) to make notification
```

Comment:
You can leave the condValue list empty (or even omit it) to make a notification trigger on any entity attribute change (regardless of the name of the attribute)